src/JobDbRepository.ts

```diff
@@ -208,7 +219,7 @@ export class JobDbRepository {
 
 	private processDbResult<DATA = unknown | void>(
 		job: Job<DATA>,
-		res: IJobParameters<DATA>
+		res?: IJobParameters<DATA>
```

Comment:
is there a way wjere res is undefined?

Reply:
yes, according to line 218, we check if res is truthy, indicating that res is actually in original agendaJS an optional parameter

Reply:
yeah right, even though I think this could not really happen in normal circumstances..